libs/barista-components/switch/src/switch.scss

```diff
@@ -3,7 +3,50 @@
 @import '../../core/src/style/transition';
 @import '../../core/src/theming/theming';
```

Comment:
We can also drop this import, as we no longer need to call the `dt-apply-theme()` mixin.
```suggestion
```

Reply:
Removed, thanks 👍